### PR TITLE
Bug 1434657 Updated Default Scheduler Policy

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1434657

### CODE REVIEW

**Comment by jeremyeder** [comment]
@ravisantoshgudimetla do you want to take a crack at this?  See the BZ linked above for context, several customer requests attached.

**Comment by ravisantoshgudimetla** [comment]
@jeremyeder Sure, I will give this a shot.

File: admin_guide/scheduler.adoc [hunk 1]
```diff
@@ -214,18 +214,26 @@ they have to be explicitly specified in the scheduler configuration file.
...
 
-. PodFitsPorts
-. PodFitsResources
 . NoDiskConflict
-. MatchNodeSelector
-. HostName
+. NoVolumeZoneConflict
+. MaxEBSVolumeCount
+. MaxGCEPDVolumeCount
+. GeneralPredicates
```
Comment (ravisantoshgudimetla) [inline admin_guide/scheduler.adoc]: I guess GeneralPredicates could be expanded to:

- noncritical Predicates(that are needed for non critical pods only. Eg: PodFitsResources).
-Essential Predicates(that are needed for every pod including critical ones: Eg: PodFitsHost, PodFitsHostPorts, PodSelectorMatches).

**Comment by bmcelvee** [comment]
@ahardin-rh please peer review

File: admin_guide/scheduler.adoc [hunk 2]
```diff
@@ -216,18 +216,27 @@ they have to be explicitly specified in the scheduler configuration file.
...
-. HostName
+. GeneralPredicates
+. PodToleratesNodeTaints
+. CheckNodeMemoryPressure
+. CheckNodeDiskPressure
 
-The default scheduler policy includes the following priority functions.
-Each of the priority function has a weight of '1' applied to it:
+The default scheduler policy includes the following priority functions. Each of
+the priority function has a weight of '1' except *NodePreferAvoidPodsPriority*,
```
Comment (ahardin-rh) [inline admin_guide/scheduler.adoc]: `1`
    `NodePreferAvoidPodsPriority`
    `1000`

Comment (ahardin-rh) [inline admin_guide/scheduler.adoc]: @bmcelvee Just some minor formatting suggestions. Otherwise, LGTM!

**Comment by bmcelvee** [comment]
[rev_history]
|xref:../admin_guide/scheduler.adoc#admin-guide-scheduler[Scheduler]
|Updated default scheduler policy in the xref:../admin_guide/scheduler.adoc#scheduler-policy[Scheduler Policy] section.
%

**Pull request merged**